README.md

```diff
@@ -7,6 +7,9 @@ The code is provided as-is with no warranties.
 > [!WARNING]  
 > As of 11.07.2024 we switched the branching model used in this chart. `main` will now contain the future oCIS 6 adjustments, while oCIS 5 will be contained in `stable-5`. Please change your chart usage accordingly.
 
+> [!WARNING]  
+> https://github.com/owncloud/ocis-charts/pull/680 introduced a breaking change in the main branch. If you are updating from an older version please ensure that you took the changes into account.
+
```

Comment:
should we have a changelog-ish file that has those warnings? I have the feeling that we'll have a very cluttered main README.md file at some point in time...

Reply:
For the time being that's better than nothing. And I hope we don't introduce too many breaking changes...